hack/patches/020-backstage_plugins_eventmesh_backend.patch

```diff
@@ -94,4 +94,4 @@ index f0d4c777d..f622a4bce 100644
 +		r.handleBackstageResources,
  		manifests.Install,
  		common.CheckDeployments,
- 		common.DeleteObsoleteResources(ctx, ke, r.installed),
+ 		common.InstallWebhookConfigs,
```

Comment:
Patch is updated to reflect changes in vendored operator code and make it applicable.